runtime/syntax/c.yaml

```diff
@@ -23,10 +23,11 @@ rules:
     - symbol.operator: "[-+*/%=<>.:;,~&|^!?]|\\b(offsetof|sizeof)\\b"
     - symbol.brackets: "[(){}]|\\[|\\]"
       # Integer Constants
-    - constant.number: "(\\b([1-9][0-9]*|0[0-7]*|0[Xx][0-9A-Fa-f]+|0[Bb][01]+)([Uu][Ll]?[Ll]?|[Ll][Ll]?[Uu]?)?\\b)"
-      # Decimal Floating Constants
```

Comment:
Why removing the indentation? The most of the comments are on the same column as definition instead of the one with the `-`.

Reply:
Whoops, that's a mistake. I will indent back.